submissions/management/commands/update_submissions_uuids.py

```diff
@@ -0,0 +1,44 @@
+"""
+Command to update all instances of old-style (hyphenated) uuid values in the
+submissions_submission table.
+
+This command takes a long time to execute, please run it on a long-lived
+background worker. The model code is resilient to both styles of uuid, this
+command just standardizes them all to be similar.
+
+EDUCATOR-1090
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from submissions.models import Submission
+
+log = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    """
+    Example usage: ./manage.py lms --settings=devstack update_submissions_uuids.py
+    """
+    help = 'Loads and saves all Submissions objects to force new non-hyphenated uuid values on disk.'
+
+    def handle(self, *args, **options):
+        """
+        By default, we're going to do this in chunks. This way, if there ends up being an error,
+        we can check log messages and continue from that point after fixing the issue.
+        """
+        START_VALUE = 0
+        CHUNK_SIZE = 1000
+        total_len = Submission.objects.count()
+        log.info("Beginning uuid update, {} rows exist in total")
+
+        current = START_VALUE;
+        while current < total_len:
+            end_chunk = current + CHUNK_SIZE
+            log.info("Updating entries {} to {}".format(current, end_chunk))
+            with transaction.atomic():
+                for submission in Submission.objects.filter(id__gte=current, id__lt=end_chunk).iterator():
+                    submission.save(update_fields=['uuid'])
+            current = current + CHUNK_SIZE
```

Comment:
I haven't included tests for this command for these reasons:

- it's difficult to get a good `setUp()` - we need lots of entries that can't be created by the ORM currently.
- this operation is safe - it will either do what it's supposed to and fix the uuids, or crash and do nothing
- we have a lot of data to fix in every environment
- this is a one-way command, it should only need to run once in each environment
- the async update command is less of a priority than restoring access to old submissions in prod

Given that, I would like to propose we push the (tested) general access code out with this first (untested) iteration of the management command. Once the time-sensitive pressures are resolved, we can test out the command while fixing the environments that need it (sandbox -> loadtest -> stage -> edge -> prod).